### DISCUSSION TITLE
Other pages of the score

### DISCUSSION THREAD

**Discussion by gerardoscoppetta** [opensheetmusicdisplay/opensheetmusicdisplay#1543] [Q&A] [answered]
I have a score that has 200 measures, I render it but I only see 54. The zoom is set to 1 and I don't want to make it smaller. How do I load other pages to see them? Is there a nextpage function or something?
Thank you
<img width="959" alt="Screenshot 2024-05-07 alle 23 31 39" src="https://github.com/opensheetmusicdisplay/opensheetmusicdisplay/assets/41266915/8e5cfbe6-6f51-4f18-b09b-3d125ba00e58">

**Comment by sschmidTU** [comment]
can you share the score (musicxml)? you can either zip it or rename it to .txt and drop it in a comment.
not sure how this could happen.

**Reply by gerardoscoppetta** [reply, marked as answer]
Hi Simon.
it was a problem of overlapping divs.
now it works fine. Thank you.
